docs/application-connector/docs/007-architecture-metadata-service.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Metadata Service
+type: Details
```

Comment:
why not arch?
also, API, same as with my other comment for connector

Reply:
Yes, the metadata of file was not updated. Tomek will handle it.